crates/router/src/types/storage/payment_method.rs

```diff
@@ -76,6 +77,19 @@ impl PaymentTokenData {
     pub fn wallet_token(payment_method_id: String) -> Self {
         Self::WalletToken(WalletTokenData { payment_method_id })
     }
+
+    pub fn get_token(&self) -> &String {
```

Comment:
@Chethan-rao - can you review this?

Reply:
In case of cards, `get_token` function doesn't really does what is says. Because cards have an object data as opposed to bank_transfer which has only one token field. I feel its better to remove this and generate token for bank transfer by explicitly matching on it